src/__tests__/validators.test.ts

```diff
@@ -241,6 +242,39 @@ describe('validateMap', () => {
   });
 });
 
+describe('validateFile', () => {
+  test('should handle json files', () => {
+    expect(validateFile('file(json)', 'testFile.json')).toEqual(true);
```

Comment:
`/testFile.json` should really be the desired input here. I realize this is going to be impacting how you built the validator. But the approach is to closely mimic the requirements for `TWILIO_VOICE_WEBHOOK_URL` and `TWILIO_SMS_WEBHOOK_URL`

Reply:
@jmulcahey-twilio Can you change this test to handle `/testFile.json` instead?